istioctl/pkg/waypoint/waypoint.go

```diff
@@ -183,6 +216,11 @@ func Cmd(ctx cli.Context) *cobra.Command {
 			return nil
 		},
 	}
+	waypointApplyCmd.PersistentFlags().StringVar(&addressType,
+		"for",
+		"service",
+		"Specify the traffic address type (service, workload, all, or none) for the waypoint.",
```

Comment:
nitty: we don't really seem to have any standard around this but mostly we don't put a period at the end of these flag descriptions.
```suggestion
		"Specify the traffic address type (service, workload, all, or none) for the waypoint",
```